charts/openshift-metering/templates/hadoop/_hadoop_config_helpers.tpl

```diff
@@ -57,8 +57,17 @@
 {{- define "core-site-xml" -}}
 <configuration>
   <property>
-      <name>fs.defaultFS</name>
-      <value>{{ .Values.hadoop.spec.config.defaultFS }}</value>
+    <name>fs.defaultFS</name>
+    <value>{{ .Values.hadoop.spec.config.defaultFS }}</value>
+  </property>
+  <property>
+    <name>fs.s3a.endpoint</name>
+    <value>https://s3.dualstack.us-east-1.amazonaws.com</value>
```

Comment:
This would lock in the region, versus probably it should come in as a region variable: 

from the doc?:
`bucketname.s3.dualstack.aws-region.amazonaws.com`